pkgs/development/libraries/gtksourceview/5.x.nix

```diff
@@ -86,10 +86,12 @@ stdenv.mkDerivation rec {
   checkPhase = ''
     runHook preCheck
 
-    XDG_DATA_DIRS="$XDG_DATA_DIRS:${shared-mime-info}/share" \
-    xvfb-run -s '-screen 0 800x600x24' dbus-run-session \
-      --config-file=${dbus}/share/dbus-1/session.conf \
-      meson test --no-rebuild --print-errorlogs
+    env \
```

Comment:
(Why) do we need to use `env`?

Reply:
It looks better when all environment variables are aligned the same.

Reply:
You could have the exact same alignment without this line! Though maybe it looks a bit weird (?)